### PR TITLE
Add Basic Unit tests suite.

### LINKED ISSUE #244: Add unit tests for plugin.
Add PHP unit testing.

### PR DESCRIPTION
**Pull request by brezocordero**
Fixes #244  .

#### Changes proposed in this Pull Request:

Basic skeleton to add unit/integration tests.

-------------------
- [x] Make sure your changes respect [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- [ ] Did you make changes, or create a **new .js file**? If **Gruntfile.js** exists in the repo, make sure to run `grunt`.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -8,5 +8,9 @@
         "exclude": [
             "!/assets"
         ]
+    },
+    "require-dev": {
+		"phpunit/phpunit": "^8.0",
```
Comment (bor0) [inline composer.json]: Please use spaces here.

File: tests/phpunit/test-class-wc-accommodation-booking.php [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+	}
+
+	/** @test - Test that instantiation of the class is working.
+	 **/
+	public function testIsAnInstanceOfWCAccommodationBooking() {
+
+		$accommodation_booking = new WC_Accommodation_Booking();
+		$this->assertInstanceOf( 'WC_Accommodation_Booking', $accommodation_booking );
+	}
+		/**
```
Comment (bor0) [inline tests/phpunit/test-class-wc-accommodation-booking.php]: Nitpick: spacing

File: composer.json (hunk 1 above)
Comment (brezocordero) [reply composer.json]: @bor0 I always use tabs to indent lines. Is it different for json files?

File: tests/phpunit/test-class-wc-accommodation-booking.php (hunk 2 above)
Comment (brezocordero) [reply tests/phpunit/test-class-wc-accommodation-booking.php]: Done!

File: composer.json (hunk 1 above)
Comment (bor0) [reply composer.json]: I think `composer.json` uses spaces. If you do `composer require X --save` it will use spaces

**Pull request merged**